### PR TITLE
hotfix: re-introducing base_url into path for req

### PR DESCRIPTION
**Pull request by mekarpeles**
Hotfix for https://github.com/internetarchive/infogami/pull/160/files#diff-7f84f23b944c54d23421925bcdbe417ae9d876a5a6395c4b5104abf7f593b364

Logins and other infogami requests broken due to missing base_url from path.

To be verified: whether the param should be url from master...mekarpeles:patch-2diff-7f84f23b944c54d23421925bcdbe417ae9d876a5a6395c4b5104abf7f593b364R125 or self.base_url + path

### CODE REVIEW

**Comment by cdrini** [comment]
Going to try patch deploying

File: infogami/infobase/client.py [hunk 1]
```diff
@@ -159,7 +159,7 @@ def request(self, sitename, path, method='GET', data=None):
         headers['X-REMOTE-IP'] = web.ctx.get('ip') or ''
 
         try:
-            response = requests.request(method, path, data=data, headers=headers)
+            response = requests.request(method, self.base_url + path, data=data, headers=headers)
```
Comment (mekarpeles) [inline infogami/infobase/client.py:162]:
```suggestion
            response = requests.request(method, f'{self.base_url}{path}', data=data, headers=headers)
```

File: infogami/infobase/client.py [hunk 2]
```diff
@@ -159,7 +159,7 @@ def request(self, sitename, path, method='GET', data=None):
         headers['X-REMOTE-IP'] = web.ctx.get('ip') or ''
 
         try:
-            response = requests.request(method, path, data=data, headers=headers)
+            response = requests.request(method, f'{self.base_url}{path}', data=data, headers=headers)
```
Comment (mekarpeles) [inline infogami/infobase/client.py:162]:
```suggestion
            response = requests.request(
                method,
                f'http://{self.base_url}{path}',
                data=data,
                headers=headers
            )
```

**Comment by cclauss** [comment]
`black infogami/infobase/client.py` to get the tests green.

**Comment by cdrini** [comment]
black changed a ton of things :/ It converted single quotes to double quotes. I thought we already blacked the repo, didn't we?

File: infogami/infobase/client.py [hunk 3]
```diff
@@ -159,7 +159,12 @@ def request(self, sitename, path, method='GET', data=None):
         headers['X-REMOTE-IP'] = web.ctx.get('ip') or ''
 
         try:
-            response = requests.request(method, path, data=data, headers=headers)
+            response = requests.request(
+                method,
+                f'http://{self.base_url}{path}',
+                data=data,
+                headers=headers
```
Comment (cdrini) [inline infogami/infobase/client.py:166]: Try to appease black

```suggestion
                method, f'http://{self.base_url}{path}', data=data, headers=headers
```

**Pull request merged**